### DISCUSSION TITLE
Feature request: Allow disable prefetch/preload & enable render functions

### DISCUSSION THREAD

**Discussion by danspratling** [nuxt/nuxt#16939] [Ideas]
For sites with many pages and/or many links prefetching creates a *lot* of additional render-blocking resource calls. In nuxt 2, it is recommended that sites with many pages disable prefetching & preloading using to prevent lots of *potential* resources from hindering performance 

However in nuxt-bridge (and nuxt 3) this feature is not available and enabling the nuxt 2 parameters does not do anything

```
export default {
  render: {
    resourceHints: false, // should disable prefetch/preload => does nothing
    asyncScripts: true, // should enable async loading on prefetched scripts => does nothing
    ... // anything passed in here also appears to not do anything
  }
}
```

For sites with many pages, this causes a blockade of resources that aren't needed being loaded before the actual content of the page.

This may be worse for sites with lots of links on the homepage (megamenus, multilingual, etc)

**Comment by leosin** [comment]
<img width="568" alt="image" src="https://user-images.githubusercontent.com/3676670/174495408-58cc614e-6849-4eb9-9a03-2f6cce385f65.png">

😭 how to disable these font prefetch..

**Comment by TheAlexLichter** [comment]
Related: #18376

**Comment by jgupta** [comment]
Configuration seems to be [available](https://github.com/nuxt/framework/blob/main/packages/schema/src/config/render.ts#L32) but not working for me.

**Comment by ProsperChiduku** [comment]
Any solution yet ?

**Comment by matthew-dean** [comment]
I'd also like to know how to disable prefetching site-wide.
